srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/EditRule.java

```diff
@@ -0,0 +1,155 @@
+
+import gov.va.med.srcalc.domain.model.Rule;
+import gov.va.med.srcalc.domain.model.ValueMatcher;
+import gov.va.med.srcalc.domain.model.Variable;
+import gov.va.med.srcalc.service.AdminService;
+import gov.va.med.srcalc.service.InvalidIdentifierException;
+
+public class EditRule
+{
+    private AdminService fAdminService;
+    private List<ValueMatcherBuilder> fMatchers;
+    private String fSummandExpression;
+    private boolean fRequired;
+    private String fDisplayName;
+    private String fNewVariableKey;
+    
+    public EditRule(final AdminService adminService) 
+    {
+        fAdminService = adminService;
+        fMatchers = new ArrayList<ValueMatcherBuilder>();
+        fSummandExpression = "";
+        fRequired = false;
+        fDisplayName = "";
+        fNewVariableKey = null;
+    }
+    
+    public EditRule(final AdminService adminService, final Rule rule)
+    {
+        // Copy the value matchers
+        fMatchers = new ArrayList<ValueMatcherBuilder>(rule.getMatchers().size());
+        for(final ValueMatcher matcher: rule.getMatchers())
+        {
+            fMatchers.add(ValueMatcherBuilder.fromPrototype(matcher));
+        }
+        fSummandExpression = rule.getSummandExpression();
+        fRequired = rule.isRequired();
+        fDisplayName = rule.getDisplayName();
+        fNewVariableKey = null;
+    }
+    
+    /**
+     * Returns the list of ValueMatcherBuilders for this EditRule.
+     */
+    public List<ValueMatcherBuilder> getMatchers()
+    {
+        return fMatchers;
+    }
+
+    public void setMatchers(final List<ValueMatcherBuilder> matchers)
+    {
+        this.fMatchers = matchers;
+    }
+
+    /**
+     * Returns the summand expression for this EditRule in String form.
+     */
+    public String getSummandExpression()
+    {
+        return fSummandExpression;
+    }
+
+    public void setSummandExpression(final String summandExpression)
+    {
+        this.fSummandExpression = summandExpression;
+    }
+
+    /**
+     * Returns true if this rule is required and false if it is not required.
+     */
+    public boolean getRequired()
+    {
+        return fRequired;
+    }
+
+    public void setRequired(final boolean required)
+    {
+        this.fRequired = required;
+    }
+
+    /**
+     * Returns this EditRule's display name.
+     */
+    public String getDisplayName()
+    {
+        return fDisplayName;
+    }
+
+    public void setDisplayName(final String displayName)
+    {
+        this.fDisplayName = displayName;
+    }
+    
+    /**
+     * Returns the variable key used to identify a new variable being added to
+     * this EditRule
+     */
+    public String getNewVariableKey()
+    {
+        return fNewVariableKey;
+    }
+    
+    public void setNewVariableKey(final String newVariableKey)
+    {
+        fNewVariableKey = newVariableKey;
+    }
+
+    /**
+     * Returns a validator for an EditRule
+     */
+    public EditRuleValidator getValidator()
```

Comment:
You can keep it if you want to, but this method is unnecessary since there is only one Validator.